test-harness/README.md

```diff
@@ -24,6 +24,16 @@ This means that you can:
 - nest test files in subdirectories
 - skip files by suffixing name with `.skip` or some other suffix.
 
+## Debugging
+
+In case some tests are giving trouble, it's possible to run the harness on the source code directly instead of the built binary. To make this happen, set `FROM_SOURCES` variable to a truthy value when running the test suite.
+
+` FROM_SOURCES=1 yarn test.harness`
+
+Keep in mind that the whole source code gets recompiled for **every** harness test. For this reason, it's recommended to use this flag with the `TESTS` env variable to narrow down the runned tests.
+
+The test is also run
```

Comment:
`also run...`
@XVincentX master of dramatic suspense :)

Reply:
I do not remember what was this for. Removed.